soh/soh/OTRGlobals.cpp

```diff
@@ -2437,9 +2437,17 @@ void SoH_ProcessDroppedFiles(std::string filePath) {
             return;
         }
 
+        CVarClearBlock(CVAR_PREFIX_SETTING);
+        CVarClearBlock(CVAR_PREFIX_CHEAT);
+        CVarClearBlock(CVAR_PREFIX_RANDOMIZER_SETTING);
+        CVarClearBlock(CVAR_PREFIX_RANDOMIZER_ENHANCEMENT);
+        CVarClearBlock(CVAR_PREFIX_COSMETIC);
+        CVarClearBlock(CVAR_PREFIX_TRACKER);
+        CVarClearBlock(CVAR_PREFIX_GENERAL);
+        CVarClearBlock(CVAR_PREFIX_REMOTE);
+        CVarClearBlock(CVAR_PREFIX_DEVELOPER_TOOLS);
```

Comment:
This is my bad for forgetting some of these blocks, but this shouldn't reset the setting, cosmetic, tracker, general and remote blocks. I think that's currently not being reset when dropping a json on the game window either.

Maybe down the line when we get more granular presets set up like Proxy has for 2ship, we could reset blocks the specific json/preset touches, but for now this is fine.